### PR TITLE
Git LFS-a-size all images

### LINKED ISSUE #3176: Switch to using git-lfs for image files
# Feature request

## Description

Our git repository is rather large because of a lot of early binaries that were shoved into source control. We've stopped doing that but we continue to shove image file formats (`*.png` for example) into our docs which frequently get refreshed. 

To prevent size creep, lets start using [git-lfs](https://git-lfs.github.com/) for all image files. This change will require all users who are contributing doc edits and/or building docs to have `git-lfs` installed first. If one is just making code changes they do not need `git-lfs` installed ... 

## User Stories

As a securedrop developer, I want to try to decrease the growing size of the securedrop repo and follow best practices by not putting non-text files in source control directly.

### PR DESCRIPTION
**Pull request by msheiny**
## Status

Work in Progress

## Description of Changes

Fixes #3176 

Changes proposed in this pull request:

* adds all image formats in docs to tracking in git-lfs
* updates documentation informing doc editors that they need to install git-lfs
* creates a requirements file for ingestion by RTD

## Testing

* Install git-lfs
* Checkout to this branch 
* `git lfs ls-files` to see LFS-enabled files
* `git lfs pull` make sure you have all the things..

Normally there will be no manual manipulation of git lfs commands after this branch gets merged in. You just use git as normal as long as its enabled in your config via `git lfs install`

## Deployment

~There is a potential weird deployment story here with RTD. Specifically in order for RTD to build this branch it will need to install a `git-lfs` dependency... which requires a change in the RTD advanced settings that will affect all branches. I'm not sure how RTD reacts when we point it to a requirements file that doesnt exist... so there is a really weird dependency loop here. I'm not sure what the best answer is but just bringing this up now as a potential blocker for this PR to be pulled in.~

RTD needs to be toggled to install our pip dependency list in order to suck in the `git-lfs` package.. testing that now

image

### If you made changes to documentation:

- [x] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

**Comment by msheiny** [comment]
Two lingering issues that need to be fixed prior to merge...

* in CI, linting needs to happen AFTER git-lfs checkout
* theres a build failure on RTD right now that i'm not 100% sure about .. that'll be more debugging time..

tl;dr - marking this PR as work in progress for now... jumping on other issues.

**Comment by msheiny** [comment]
> theres a build failure on RTD right now that i'm not 100% sure about .. that'll be more debugging time..

Just to clarify, build failure is only affecting this branch and not latest. So we will get no interruption in doc updates

**Comment by msheiny** [comment]
I'm closing this PR for now. Needs more work. A rebase and its muddlin' up the PR list.

**Comment by msheiny** [comment]
(Please leave that branch lingering though for a bit  ) :angel:
